fineract-provider/src/main/java/org/apache/fineract/useradministration/api/RolesApiResourceSwagger.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Created by sanyam on 23/8/17.
  */
-@SuppressWarnings({ "MemberName" })
+@SuppressWarnings({ "MemberName", "AbbreviationAsWordInName" })
```

Comment:
here it's probably OK, because you can't change it, as it's already part of the external Swagger API? But then it's worth to document it like this:

```suggestion
// Cannot change names of public API without breaking backwards compatibility
@SuppressWarnings({ "MemberName", "AbbreviationAsWordInName" })
```

Reply:
Yes agreed, Updating.